tests/sdk/test_session_context.py

```
@@ -0,0 +1,563 @@
+    This test covers lines 79-80 in session_context.py where assistant.id is accessed.
+    """
+    honcho_client, client_type = client_fixture
+    session_id = "test-session-context"
+
+    # Create test messages with different peer IDs
+    messages = [
+        Message(
+            id="msg1",
+            content="Hello from user",
+            peer_id="user123",
+            created_at=datetime.now(),
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)